### ISSUE TITLE
Expand class on <button> causes duplicate <a> tag

### ISSUE THREAD

**Issue by rickysullivan** [zurb/inky#13]
```
<button class="expand" href="http://zurb.com">Button</button>
```

This is causing 

```
<table class="button expand">
  <tr>
    <td>
      <table>
        <tr>
          <td>
            <center>
              <a href="http://zurb.com">
                <a href="http://zurb.com">Button</a>
              </a>
            </center>
          </td>
        </tr>
      </table>
    </td>
  </tr>
</table>
```

**Comment by gakimball** [comment]
Fixed via 16c3bad3d2e05816f410cb5e0eefc06082fe195f

Just published 1.0.0 which has this fix.

**Comment by rickysullivan** [comment]
Thanks.
